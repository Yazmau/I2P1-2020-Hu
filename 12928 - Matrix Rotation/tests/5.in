5 5
398630021 564718673 412106895 373282330 667584659
79866982 365329221 564718673 79866982 365329221
667584659 291882089 753287868 79866982 690430685
632939232 753287868 79866982 657489855 358502890
398630021 690430685 398630021 398630021 667584659
101
