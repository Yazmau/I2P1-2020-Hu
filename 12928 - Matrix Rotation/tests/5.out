398630021 632939232 667584659 79866982 398630021
690430685 753287868 291882089 365329221 564718673
398630021 79866982 753287868 564718673 412106895
398630021 657489855 79866982 79866982 373282330
667584659 358502890 690430685 365329221 667584659
